deploy/helm/kubecf/templates/ops.yaml

```diff
@@ -20,8 +20,9 @@ data:
 
 {{- $root := . }}
 
-{{ include "kubecf.ops" (dict "Root" $root "Path" "assets/operations/unordered/move_auctioneer.yaml") }}
-{{ include "kubecf.ops" (dict "Root" $root "Path" "assets/operations/unordered/move_routing_api.yaml") }}
+{{- range $path, $bytes := .Files.Glob "assets/operations/job_moving/*" }}
+{{ include "kubecf.ops" (dict "Root" $root "Path" $path) }}
```

Comment:
We never use `$bytes` here either.

Reply:
Fixed in https://github.com/cloudfoundry-incubator/kubecf/pull/619/commits/49c5c179b823834d7b592b1d83725f079dc3312f.